examples/aishell/s1/README.md

```diff
@@ -0,0 +1,19 @@
+# Performance Record
```

Comment:
update readme: https://github.com/mobvoi/wenet/pull/41/files

Reply:
will update this file when more experiments are done.